src/components/UserInfo.js

```diff
@@ -24,8 +24,7 @@ export default class UserInfo {
   setUserInfo({name, about, avatar}) {
     this.#profileUsername.textContent = name;
     this.#profileAbout.textContent = about;
-    this.#avatar.src = avatar;
-    this.#avatar.alt = name;
+    this.setAvatar({avatar});
   }
 
   setAvatar({avatar}) {
```

Comment:
```js

this.#avatar.alt = this.#profileUsername;
```


You haven't corrected my remark https://github.com/rstaats0925/se_project_aroundtheus/pull/1#discussion_r1243284795